samples/sample_multi_transcode/src/transcode_utils.cpp

```diff
@@ -2102,6 +2103,10 @@ mfxStatus CmdProcessor::ParseParamsForOneSession(mfxU32 argc, msdk_char *argv[])
         {
             InputParams.bUseOpaqueMemory = false;
         }
+        else if (0 == msdk_strcmp(argv[i], MSDK_STRING("-opaq")))
+        {
+            InputParams.bUseOpaqueMemory = false;
```

Comment:
May be InputParams.bUseOpaqueMemory = true; ?

Reply:
thanks, Roma!